src/app/components/autocomplete-dropdown/autocomplete-dropdown.component.scss

```diff
@@ -17,12 +17,9 @@ $disabled-background-color: #eeeeee;
       right: 10px;
       top: calc(50% - 8px);
       font-size: 8px;
-      background-color: $default-bg-color;
+      background-color: unset;
```

Comment:
Cool! Haven't seen this property before, seems like a quite complicated property 😅 
> ... is a combination of the initial and inherit keywords...
> -- https://developer.mozilla.org/en/docs/Web/CSS/unset